apps/mull-ui/src/app/components/event-bullet/event-bullet.scss

```diff
@@ -0,0 +1,33 @@
+.event-bullet-container {
+  display: flex;
+  align-items: center;
+  justify-content: left;
+  position: relative;
+  padding: 1rem 0;
+}
+
+.event-photo {
+  max-width: 3rem;
```

Comment:
is it possible to match the height of the date-box? having 4rem does seems to work

Reply:
![image](https://user-images.githubusercontent.com/35787279/109536564-e078fb80-7a8b-11eb-9500-659901907f57.png)
Height adjusted